### PR TITLE
[PXCT-148]add updater info

### PR DESCRIPTION
**Pull request by g-castronovo**
## What This PR Changes
- Add paragraph about the updater

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md [hunk 1]
```diff
@@ -27,9 +27,26 @@ Your Alvik robot is equipped with three ready-to-go examples. To choose one of t
...
+6. Once connected, click the "Update" button and wait for the process to complete.
+
 ### Let's Start Coding Alvik
 
-Alvik is intended to be programmed with MicroPyton. We recommend you to install the [Arduino Lab for MicroPython](https://labs.arduino.cc/en/labs/micropython) editor. Follow the instructions next to the download link to install it and open the IDE.
+Alvik is intended to be programmed with MicroPython. We recommend you to install the [Arduino Lab for MicroPython](https://labs.arduino.cc/en/labs/micropython) editor. Follow the instructions next to the download link to install it and open the IDE.
+
+If you're using a **Chromebook**, there is no installable version of the software available. Instead, you'll use the [**online version**](https://lab-micropython.arduino.cc/) using Chrome. You'll need to login with an Arduino account to use it. 
+
+![Online Lab Micropython](/explore-robotics-micropython/_assets/projects/00/online-lab-micropython.jpg)
```
Comment (pedromsousalima) [inline content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md:49]:
```suggestion
![Online Lab Micropython](assets/online-lab-micropython.jpg)
```

File: content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md [hunk 2]
```diff
@@ -108,6 +125,6 @@ You are now all set, disconnect Alvik from the computer, put some obstacles arou
 
 ### Next Steps
 
-* There is a set of already built examples that will help you to better understand how Alvik works, you can download them from [this link](https://github.com/arduino/arduino-alvik-mpy/archive/refs/tags/0.2.0.zip), unzip them in your already created _alvik_ folder and you will be able to see them straight away in the Arduino Labs for MicroPython IDE.
```
Comment (pedromsousalima) [inline content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md:111]: Is there a specific reason why this particular version is used? This looks like a old version https://github.com/arduino/arduino-alvik-mpy/releases

File: content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md (hunk 1 above)
Comment (pedromsousalima) [reply content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md:49]: Does this fix the linter?

**Review by pedromsousalima — COMMENTED** [review]
Looks good my comments are mostly questions :)

File: content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md (hunk 2 above)
Comment (pedromsousalima) [reply content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md:111]: @g-castronovo If new versions are compatible we might benefit from not linking directly the download and instead send to the release pages as new versions will render the content outdated :) Will still work but they will be on a different verison

**Review by pedromsousalima — APPROVED** [review]
LGTM

**Review by jcarolinares — APPROVED** [review]
Review it too, all fine

**Pull request merged**